README.md

```diff
@@ -315,7 +315,7 @@ module.exports = {
         test: /\.lazy\.css$/i,
         use: [
           { loader: 'style-loader', options: { injectType: 'lazyStyleTag' } },
-          'css-loader',
+          { loader: 'css-loader', options: { modules: true } },
```

Comment:
We don't need `modules: true` in all examples, only where we say about modules

Reply:
All examples for `injectType` option contains accessing for css class names (via object itself or via `locals` property). As i can see, class names will be accessible only if `modules` set to `true`.

In case of 
```js
use: [
  { loader: 'style-loader' },
  { loader: 'css-loader' }
]
```

Result of

```js
import css1 from './1.css'
console.log(css1)
```

will be 
```js
{}
```

Another option value:
```js
use: [
  { loader: 'style-loader', options: { injectType: 'lazyStyleTag' } },
  { loader: 'css-loader' }
]
```

Result is:
```js
{ use: ƒ, unuse: ƒ }
```

There is no class names in object, nor locals property.

But if i set `modules: true` result will be:
```js
// 1st example, selector is a class name
{ selector: "tztrC7Uy9NarvcRmBTE5 _3-ZFXfyURwvBq_mDbTewjd" }

// 2nd example
{
  locals: { selector: "tztrC7Uy9NarvcRmBTE5 _3-ZFXfyURwvBq_mDbTewjd" }
  use: ƒ
  unuse: ƒ
}
```

Reply:
So i thought examples without explicit `modules: true` little bit confusing in parts with getting class names. Or may be there is a bug?

Anyway, i could remove `modules: true` for other `injectType` values.